src/panel/app_panel.js

```diff
@@ -254,12 +268,7 @@ export default class AppPanel {
     this.activePoiId = null;
     this.panels
       .filter(panel => panel !== panelToOpen)
-      .forEach(panel => {
-        if (panel === this.categoryPanel) {
```

Comment:
If you remove this test, the category markers won't get removed after leaving the category panel. Instead of removing it, I think you need to adapt it for the `eventListPanel` instead.

Reply:
indeed, I think a bad manip made me remove these lines. Fixed :)